README.md

```diff
@@ -1,134 +1,77 @@
-- Check poetry by just typing `poetry` into terminal, it should recognize and list options etc
+3. Now at the project root you can run `poetry install` which will install all packages required for now
+  - To double check everything is working as intended, navigate to `src/sinr/models.py` write a function that prints something. Now, navigate to `src/backend/app` and create a temporary python file that you will delete later, in the file enter `from sinr.models import *`, then call the function you just created from that file, make sure it prints correctly and does not give errors.
 
-- Now at the project root you can run `poetry install` which will install all packages required for now
-- To double check everything is working as intended, navigate to `src/sinr/models.py` write a function that prints something. Now, navigate to `src/backend/app` and create a temporary python file that you will delete later, in the file enter `from sinr.models import *`, then call the function you just created from that file, make sure it prints correctly and does not give errors.
-
-### Running React App and Server FastAPI
-- You run server first, then application
-- Open two terminals.
-- To start the server, navigate to project root, run `uvicorn src.backend.app.main:app --reload`, to make sure it is working go to `http://localhost:8000/hello/` in your browser
-- To start the application, navigate to `src/frontend/` you see there are js things, you are at the right spot. Now run `npm start`, it will start running the app and should automatically open to page in your browser.
-
-# Old instructions, don't read further
-
-## Installing Dependencies and Packages
-Use these steps for setting up a development environment to install and work with code in this template:
-1) Set up a Python 3 virtual environment using [Conda](https://docs.conda.io/projects/conda/en/latest/user-guide/install/index.html#) or [Virtualenv](https://virtualenv.pypa.io/en/latest/index.html). Read [Python Virtual Environments: A Primer](https://realpython.com/python-virtual-environments-a-primer/#the-virtualenv-project) for details on how to get started with virtual environments and why you need them.
-2) Activate your virtual environment.
-
-3) Install the package.
-	- If you want to just use the scripts and package features, install the project by running `pip install .` from the root directory.
-	- If you will be changing the code and running tests, you can install it by running `pip install -e .[test,dev]`. The `-e/--editable` flag means local changes to the project code will always be available with the package is imported. You wouldn't use this in production, but it's useful for development.
-
-
-For example, if you use Conda, you would run the following to create an environment named `template` with python version 3.10, then activate it and install the package in developer mode:
+## :penguin: Starting the app
+#### Activate the inatator environment:
+```bash
+ conda activate inatator
 ```
```

Comment:
Refer to data README, it is not clear that we need to get the data folder and pertained models, and where to find the instructions for this.